packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.styles.ts

```diff
@@ -99,7 +89,7 @@ export const getStyles = (props: IDetailsRowStyleProps): IDetailsRowStyles => {
   const shimmerRightBorderStyle = `${cellStyleProps.cellRightPadding * 4}px solid ${colors.defaultBackgroundColor}`;
   const shimmerLeftBorderStyle = `${cellStyleProps.cellLeftPadding}px solid ${colors.defaultBackgroundColor}`;
   const selectedStyles: IStyle = [
-    getFocusStyle(theme, -1, undefined, undefined, themePrimary, white),
+    getFocusStyle(theme, -1, undefined, undefined, neutralSecondary, white),
```

Comment:
Should we not use the focusColor defined in semanticColors?

Reply:
Fixed.